app/main.py

```diff
@@ -1,3 +1,81 @@
-def shop_trip():
-    # write your code here
-    pass
+import datetime
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def calculate_distance(
+        location1: list,
+        location2: list
+) -> float:
+    return ((location1[0] - location2[0])**2
+            + (location1[1] - location2[1])**2)**0.5
+
+
+def calculate_fuel_cost(
+        distance: float,
+        fuel_price: float,
+        fuel_consumption: float
+) -> float:
+    return (distance / 100) * fuel_consumption * fuel_price
+
+
+def shop_trip() -> None:
```

Comment:
it is really big function. try to seperate it to a few functions/methods with different responsibilities